contracts/lib/NonMerklizedIssuer.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity 0.8.16;
+
+import {SmtLib} from "../lib/SmtLib.sol";
+import {IdentityLib} from "../lib/IdentityLib.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @dev INonMerklizedIssuer. Interface for non-merklized issuer
+ */
+interface INonMerklizedIssuer {
+    /**
+     * @dev listUserCredentialIds. Get list of user credentials identifiers
+     * @param _userId user id
+     */
+    function listUserCredentialIds(uint256 _userId) external returns (uint256[] memory);
+
+    /**
+     * @dev getCredential. Get credential by user id and credential id.
+     * Returns credential data, core claim and subject fields.
+     */
+    function getCredential(
+        uint256 _userId,
+        uint256 _credentialId
+    )
+        external
+        returns (
+            NonMerklizedIssuerLib.CredentialData memory,
+            uint256[8] memory,
+            NonMerklizedIssuerLib.SubjectField[] memory
+        );
+
+    /**
+     * @dev credentialProtocolVersion. Get version of the protocol
+     */
+    function credentialProtocolVersion() external returns (string memory);
+}
+
+/**
+ * @dev NonMerklizedIssuer. Non-merklized issuer types
+ */
+library NonMerklizedIssuerLib {
```

Comment:
Why do you need this lib which implements no logic? May be it's better move all the structs into the `INonMerklizedIssuer` definition?

Reply:
We want to have a public constant to represent version lib.  CREDENTIAL_PROTOCOL_VERSION
Is possible for interface?

Reply:
You can implement `MerklizedIssuer` class which implements `INonMerklizedIssuer` interface